platform/ios/src/MGLMapView.mm

```diff
@@ -1159,6 +1159,10 @@ - (void)touchesBegan:(__unused NS_SET_OF(UITouch *) *)touches withEvent:(__unuse
 {
     _changeDelimiterSuppressionDepth = 0;
     _mbglMap->setGestureInProgress(false);
+    if (self.userTrackingState == MGLUserTrackingStateBegan)
+    {
+        [self setUserTrackingMode:MGLUserTrackingModeNone];
```

Comment:
The only difference between this change and the one in #7916 is the `animated` parameter. Is it important to keep animation on in this case?